data/json/items/ammo.json

```diff
@@ -422,7 +422,7 @@
         "volume" : 1,
         "weight" : 11,
         "ammo_type" : "12mm",
-        "damage" : 45,
```

Comment:
You could consider migrating the `ammo` to the correct subdir and linting it as well